brush/brush.go

```diff
@@ -39,26 +39,28 @@ func (b Brush) UUID() uuid.UUID {
 
 var bb = cube.Box(-0.125, -0.125, -0.125, 0.125, 0.125, 0.125)
 
-func (b Brush) Use(p *player.Player) {
+func (b Brush) Use(p *player.Player, h *world.EntityHandle) {
```

Comment:
This is still not right. `Brush.Use()` is being called on another goroutine which invalidates `*player.Player`s. Don't pass the player to this function, only pass the handle. You can retrieve the player in the transaction by asserting `e`: `p := e.(*player.Player)`